### DISCUSSION TITLE
how to use set_local_address in hyper 1 ?

### DISCUSSION THREAD

**Discussion by i18nsite** [hyperium/hyper#3516] [Q&A]
there is https://github.com/hyperium/hyper/pull/2172/files set_local_address before, but not in hyper 1

**Comment by seanmonstar** [comment]
If you're using the legacy client in hyper-util, the method is here: https://docs.rs/hyper-util/latest/hyper_util/client/legacy/connect/struct.HttpConnector.html#method.set_local_address
